### PR TITLE
added socialprojects information

### PR DESCRIPTION
**Pull request by mojoaxel**

### CODE REVIEW

File: communitys_franken.json [hunk 1]
```diff
@@ -251,7 +271,10 @@
 			"nodeMaps" : [{
 				"url" : "https://monitoring.freifunk-franken.de/map?mapcenter=49.13304,10.07652,14",
 				"mapType" : "geographical"
-			}]
+			}],
+			"socialprojects": {
+				"number": 2
+			},
```
Comment (kratz00) [inline communitys_franken.json]: This comma breaks the JSON.
Did you test or validate before creating this pull request?

Comment (mojoaxel) [inline communitys_franken.json]: I could not test it because of python dependency errors.

Comment (kratz00) [inline communitys_franken.json]: IIRC the script has no external dependencies, everything should be covered in Python 3.
Besides validating the JSON format can be done with any other JSON parser/validator.

**Comment by kratz00** [comment]
Looks good now, please squash your commits, then I will merge your pull request.

File: communitys_franken.json (hunk 1 above)
Comment (mojoaxel) [inline communitys_franken.json]: Yes, sorry that was my mistake. No external dependencies needed.

**Comment by mojoaxel** [comment]
@kratz00 for your info: github provides [squash merges](https://github.com/blog/2141-squash-your-commits) since a few month.

**Pull request merged**